### PR TITLE
Sraithel k cpython

### PR DESCRIPTION
**Pull request by swraithel**
Changing Kansas City Scraping to python with associated CSV's. Removed old R script and CSV's. Should match the format of the Baltimore script.

### CODE REVIEW

File: scrapers/src/crime/kc/data/kc-2017-crime.csv [hunk 1]
```diff
@@ -0,0 +1,7308 @@
...
+2017,1,1,Possession/Sale/Dist,3
+2017,1,1,Property Damage,28
+2017,1,1,Resisting an Officer,3
+2017,1,1,Sex Off Fondle - mol,1
+2017,1,1,Statutory Rape,1
+2017,1,1,Stealing All Other,5
+2017,1,1,Stealing Auto Parts/,3
+2017,1,1,Stealing From Auto,7
+2017,1,1,Stealing Shoplifting,5
+2017,1,1,Stealing from Buildi,4
```
Comment (devinaconley) [inline scrapers/src/crime/kc/data/kc-2017-crime.csv]: Looks like a bunch of the descriptions are cut off. Is that coming from our end or the data source?

Comment (devinaconley) [inline scrapers/src/crime/kc/data/kc-2017-crime.csv]: Looks like a bunch of the descriptions are cut off. Is that coming from our end or the data source?

Comment (devinaconley) [inline scrapers/src/crime/kc/data/kc-2017-crime.csv]: Looks like a bunch of the descriptions are cut off. Is that coming from our end or the data source?

Comment (swraithel) [inline scrapers/src/crime/kc/data/kc-2017-crime.csv]: Ah, good catch. It looks like it's on the data source end. Here's a view of the original source. https://data.kcmo.org/Crime/KCPD-Crime-Data-2016/wbz8-pdv7/data
They do have an offense code and IBRS code which might be helpful for cleaning them up?
It looks like the description field may cut off around 20ish characters. Have we seen this any any of the other crime datasets?

Comment (devinaconley) [inline scrapers/src/crime/kc/data/kc-2017-crime.csv]: Cool, let's just consider that part of the eventual merge then

**Pull request merged**